pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -128,41 +142,15 @@ sub _process_template {
     die "Template processing failed: $output";
 }
 
-sub create_purchase_link {
-    my ($self) = @_;
-
-    my $imunify    = Whostmgr::Imunify360->new();
-    my $installed  = $imunify->is_product_installed();
-    my $custom_url = $imunify->get_custom_url();
-    my $price      = $imunify->get_product_price();
-    my $cp_url     = $self->base_path('scripts12/purchase_imunify360_init');
-
-    if ($custom_url) {
-        return locale()->maketext( '[output,url,_1,Get Imunify360,_2,_3].', $custom_url, 'target', '_blank', );
-    }
-    if ($installed) {
-        return locale()->maketext( 'To purchase a license, visit the [output,url,_1,cPanel Store,_2,_3].', $cp_url, 'target', '_parent', );
-    }
-    if ($price) {
-        return locale()->maketext( '[output,url,_1,Get Imunify360,_2,_3] for $[_4]/month.', $cp_url, 'target', '_parent', $price );
-    }
-    return locale()->maketext( '[output,url,_1,Get Imunify360,_2,_3].', $cp_url, 'target', '_parent', );
-}
-
 sub _suggest_imunify360 {
     my ($self) = @_;
 
-    my $imunify              = Whostmgr::Imunify360->new();
```

Comment:
Instead of deleting all these variables that are being used in the if statements.
Could we do something like this? So the refactor is minimal
```
my $licensed = Whostmgr::Imunify360::is_imunify360_licensed();
my $installed = Whostmgr::Imunify360::is_imunify360_installed();
my $is_kernelcare_needed = Whostmgr::Imunify360::get_kernelcare_data()->{'disabled'};
```

Reply:
This was a straight revert of the changes to the module, so that's why it has some deletions that might not be strictly necessary. In a way, I think it's safer to restore it to the exact state it was in that was known to be v80-compatible, even if it makes for a larger diff. We can work on it some more before submitting it upstream if you'd like, though.

Reply:
I did not know was a revert. I tested in my sandbox and everything works.